build.sbt

```diff
@@ -18,6 +20,6 @@ libraryDependencies ++= Seq(
 
 libraryDependencies ++= Seq(
   "io.circe" %% "circe-yaml" % "0.5.0",
-  "org.eclipse.jgit" % "org.eclipse.jgit" % "4.6.1.201703071140-r",
+  "org.eclipse.jgit" % "org.eclipse.jgit" % "4.6.1.201703071140-r" exclude("org.eclipse.jgit", "org.eclipse.jgit"),
```

Comment:
Instead we use a custom build of jgit that eliminates (lazifies) some Threads in statics

Reply:
https://github.com/eclipse/jgit/pull/65